neural_lam/models/hi_lam.py

```diff
@@ -1,16 +1,17 @@
 # Third-party
 from torch import nn
 
-# First-party
-from neural_lam.interaction_net import InteractionNet
-from neural_lam.models.base_hi_graph_model import BaseHiGraphModel
+# Local
+from ..interaction_net import InteractionNet
+from .base_hi_graph_model import BaseHiGraphModel
 
 
 class HiLAM(BaseHiGraphModel):
-    """
-    Hierarchical graph model with message passing that goes sequentially down
-    and up the hierarchy during processing.
+    """Hierarchical graph model with message passing that goes sequentially down and up
+    the hierarchy during processing.
+
     The Hi-LAM model from Oskarsson et al. (2023)
+
     """
 
     def __init__(self, args):
```

Comment:
Here the following two args are missing
```suggestion
def __init__(self, args, datastore, forcing_window_size):
        super().__init__(args, datastore, forcing_window_size)
```

Reply:
Fixed in https://github.com/leifdenby/neural-lam/commit/d4d438ff372ce6568c62717c4bd857de83f888a5